include/cantera/base/global.h

```diff
@@ -102,37 +93,56 @@ void appdelete();
 //! @copydoc Application::thread_complete
 void thread_complete();
 
-//! Returns the Cantera version. This function when needing to access the version from a
-//! library, rather than the `CANTERA_VERSION` macro that is available at compile time.
+//! @defgroup globalSettings  Global Cantera Settings
+//! @brief Functions for accessing global %Cantera settings.
+//! @ingroup globalData
+
+//! @addtogroup globalSettings
+//! @{
+
+//! Returns `true` if %Cantera was loaded as a shared library in the current
+//! application. Returns `false` if it was statically linked.
+//! @since New in Cantera 3.0
+bool usingSharedLibrary();
+
+//! @name Cantera Version Information
+//! @{
+
+//! Returns the %Cantera version. This function is used to access the version from a
+//! library, rather than the \c CANTERA_VERSION macro that is available at compile time.
 //! @since New in Cantera 3.0
 string version();
 
-//! Returns the hash of the git commit from which Cantera was compiled, if known
+//! Returns the hash of the git commit from which %Cantera was compiled, if known
 std::string gitCommit();
 
-//! Returns true if Cantera was compiled in debug mode. Used for handling some cases
+//! @}
+
+//! Returns true if %Cantera was compiled in debug mode. Used for handling some cases
 //! where behavior tested in the test suite changes depending on whether the `NDEBUG`
 //! preprocessor macro is defined.
 bool debugModeEnabled();
 
-//! Returns true if Cantera was compiled with C++ HDF5 support.
+//! Returns true if %Cantera was compiled with C++ \c HDF5 support.
 //! @since  New in Cantera 3.0.
 bool usesHDF5();
 
+//! @}
+
 /*!
- * @defgroup logs Diagnostic Output
+ * @defgroup textlogs Diagnostic Output
+ * @brief Logging and generation of diagnostic output.
  *
  * Writing diagnostic information to the screen or to a file. It is often
  * useful to be able to write diagnostic messages to the screen or to a file.
- * Cantera a set of procedures for this purpose designed to write text messages
+ * Cantera defines a set of procedures for this purpose designed to write text messages
  * to the screen to document the progress of a complex calculation, such as a
  * flame simulation.
+ * @ingroup exceptions
```

Comment:
The hierarchy here seems a bit off to me -- logging isn't just about errors. Would the following make more sense?
- Errors & Diagnostics
  - Exceptions
  - Warnings
  - Logging